gulp-tasks/build-packages.js

```diff
@@ -31,11 +32,12 @@ const ERROR_NO_NAMSPACE = oneLine`
 
 const buildPackage = (packagePath, buildType) => {
   const packageName = pkgPathToName(packagePath);
-  const browserBundlePath = path.join(packagePath, 'browser.mjs');
+  const browserEntry = path.join(packagePath, constants.PACKAGE_BUILD_DIRNAME,
+    constants.BROWSER_ENTRY_FILENAME);
 
   // First check if the bundle file exists, if it doesn't
   // there is nothing to build
-  if (!fs.pathExistsSync(browserBundlePath)) {
+  if (!fs.pathExistsSync(browserEntry)) {
```

Comment:
It's unclear from the name `browserEntry` what this is expected to be. I assume as you're checking `pathExistsSync(...)` that this could perhaps be `browserEntryPath`. Anything to improve readability would be appreciated.